os/fs/driver/mtd/smart.c

```diff
@@ -2258,10 +2257,10 @@ static int smart_scan(FAR struct smart_struct_s *dev)
 	/* Validate the sector is valid ... may be an unformatted device. */
 
 	if (sector != 0xFFFF) {
-		/* Read the sector data. */
-
-		ret = MTD_BREAD(dev->mtd, sector * dev->mtdBlksPerSector, dev->mtdBlksPerSector, (uint8_t *)dev->rwbuffer);
-		if (ret != dev->mtdBlksPerSector) {
+		/* Read the sector data from start to offset of SMART_WEAR_LEVEL_FORMAT_SIG. */
```

Comment:
Does `SMART_WEAR_LEVEL_FORMAT_SIG` indicate a size value? It is difficult to understand from the name. Can we change the name accordingly pls?

Reply:
It means some specific offset within sector.